index.js

```diff
@@ -1,19 +1,19 @@
-const fs = require('fs');
-const glob = require('glob');
-const path = require('path');
-const log = require('loglevel');
-const prettier = require('prettier');
+import fs from 'fs';
+import glob from 'glob';
+import path from 'path';
+import {build} from 'documentation';
+import log from 'loglevel';
+import prettier from 'prettier';
 
-const {makeParser} = require('./src/parsers');
+import {makeParser} from './src/parsers.js';
```

Comment:
same here.

Reply:
It's one of the differences between ES modules and CommonJS.

https://nodejs.org/docs/latest-v12.x/api/esm.html#esm_mandatory_file_extensions
> A file extension must be provided when using the import keyword. Directory indexes (e.g. './startup/index.js') must also be fully specified.